src/main/java/Duke.java

```diff
@@ -1,10 +1,116 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
+    public  Duke(){
+
+    }
+    ArrayList<Task> tasks = new ArrayList<>();
+    private final static String horLine = "------------------------------------------------------";
+    private final static String welcomeMsg = horLine + "\nHello! I'm Duke\n" +
+            "What can I do for you?\n" + horLine;
+
+    private void addToList(String taskName) {
+        try{
+            //if task is a todo
+            if (taskName.matches("\\btodo\\s.*\\b")){
+                Task newTask = new ToDo(taskName.substring(5));
+                tasks.add(newTask);
+                System.out.println(horLine + "\n\tGot it. I've added this task:\n" + "\t" + newTask.toString());
+                System.out.println("\tNow you have " + tasks.size() + " tasks in the list." + "\n" + horLine);
+
+            }
+            //if task is an event
+            else if (taskName.matches("\\bevent\\s.*\\s/at\\s.*\\b")){
+                String des = taskName.substring(6,taskName.indexOf("/")-1);
+                String at = taskName.substring(taskName.indexOf("/")+4,taskName.length());
+                Task newTask = new Event(des,at);
+                tasks.add(newTask);
+                System.out.println(horLine + "\n\tGot it. I've added this task:\n" + "\t" + newTask.toString());
+                System.out.println("\tNow you have " + tasks.size() + " tasks in the list." + "\n" + horLine);
+
+            }
+            ////if task is a deadline
```

Comment:
```suggestion
            //if task is a deadline
```
I think you might have added additional `/` by accident

Reply:
oh yes, thank you for pointing it out.